### PR TITLE
Dolev viz

### PR DESCRIPTION
**Pull request by DolevArtzi**

### CODE REVIEW

**Comment by DolevArtzi** [comment]
I left in two different (but related) `console.log` statements https://github.com/mitmedialab/prg-extension-boilerplate/blob/311c955b6262b26a1802e4bc588f4c88cd76926e/packages/scratch-vm/src/extensions/scratch3_musiccreation/sheetmusic.js#L912 https://github.com/mitmedialab/prg-extension-boilerplate/blob/311c955b6262b26a1802e4bc588f4c88cd76926e/packages/scratch-vm/src/extensions/scratch3_musiccreation/sheetmusic.js#L921

These are important because they show when a note is adjusted for the purposes of displaying sheet music (see https://github.com/mitmedialab/prg-extension-boilerplate/issues/103). Also note that the several `log.log` statements in `sheetmusic.js` were there before I started working on this code. Not sure if we still want them.

**Comment by pmalacho-mit** [comment]
Looking great, Dolev!I approve (but I'm sure we'll talk about some of my comments in our standup)!

File: packages/scratch-vm/src/extensions/scratch3_musiccreation/vizhelpers.js [hunk 1]
```diff
@@ -180,6 +188,64 @@ class VizHelpers {
         return penState;
     }
 
+    toggleVisMode (args,util) {
+        let status = Cast.toNumber(args.STATUS);
+        let mode = Cast.toNumber(args.FORMAT);
+        this._visState['mode'] = mode;
```
[the remark this answers is not in the document]
Comment (pmalacho-mit) [reply packages/scratch-vm/src/extensions/scratch3_musiccreation/vizhelpers.js:194]: this.visState = {mode, status};

[the remark this answers is not in the document]
Comment (pmalacho-mit) [reply packages/scratch-vm/src/extensions/scratch3_musiccreation/vizhelpers.js:194]: You'll mostly see the reverse: `const {mode, status} = this.visState;`

**Pull request merged**